### ISSUE TITLE
manifest.json not found

### ISSUE THREAD

**Issue by sebzur** [scoutapp/scout_apm_python#22]
Hi,

I tried to run Scout with my django software. After `pip instal scout-apm` followed by default config: `scout_apm.django` added to INSTALLED_APPS and SCOUT_[MONITOR|KEY|NAME] settings configured the service refuses to start. Scout complains that it can not find `manifest.json` file.

```
IOError: [Errno 2] No such file or directory: '/tmp/scout_apm_core/scout_apm_core-latest-x86_64-unknown-linux-gnu/manifest.json'
```

I have scout installed within the virtualenv. The whole error stack is attached below. Any hints?

```bash
Traceback (most recent call last):
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
    worker.init_process()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 126, in init_process
    self.load_wsgi()  
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/gunicorn/util.py", line 352, in import_app
    __import__(module)
  File "/home/username/PRODUCTION/compose/wsgi.py", line 14, in <module>
    application = get_wsgi_application()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/django/core/wsgi.py", line 14, in get_wsgi_application
    django.setup()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/django/apps/registry.py", line 115, in populate
    app_config.ready()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/scout_apm/django/apps.py", line 26, in ready
    installed = scout_apm.core.install()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/scout_apm/core/__init__.py", line 20, in install
    CoreAgentManager().launch()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/scout_apm/core/core_agent_manager.py", line 37, in launch
    if self.verify() is not True:
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/scout_apm/core/core_agent_manager.py", line 100, in verify
    manifest = CoreAgentManifest(self.core_agent_dir + '/manifest.json')
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/scout_apm/core/core_agent_manager.py", line 202, in __init__
    self.parse()
  File "/home/username/.virtualenvs/django1.8/local/lib/python2.7/site-packages/scout_apm/core/core_agent_manager.py", line 209, in parse
    with open(self.manifest_path) as manifest_file:
IOError: [Errno 2] No such file or directory: '/tmp/scout_apm_core/scout_apm_core-latest-x86_64-unknown-linux-gnu/manifest.json'
```

**Comment by cschneid** [comment]
@sebzur - at launch, we download the binary portion of our agent, and place it in that /tmp/ location. The `manifest.json` file describes the download.

Can you do an `ls` of the `/tmp/scout_apm_core/scout_apm_core-latest-x86_64-unknown-linux-gnu/` directory? It should contain a `.tgz` file, which in turn has 2 files inside of it (binary `core-agent` and the `manifest.json`).

**Comment by cschneid** [comment]
@sebzur I just released agent version 1.1.1. It catches that IOError correctly now.

Then, if you turn up logging ( http://help.apm.scoutapp.com/#python-logging ), we can figure out why it isn't downloading or unpacking correctly.

**Comment by cschneid** [comment]
Hi @sebzur, you're right - we switched to supporting Django 1.10+ in this first beta phase of the agent.

We'll expand out to supporting 1.8+ pretty quickly, but the instrumentation approach we ended up on relies on the revamped middleware from 1.10+ for the moment.

I just created a dedicated issue for that support #26.
